gcloud/credentials.py

```diff
@@ -294,15 +294,22 @@ def _get_expiration_seconds(expiration):
 def generate_signed_url(credentials, resource, expiration,
                         api_access_endpoint='',
                         method='GET', content_md5=None,
-                        content_type=None):
+                        content_type=None, response_type=None,
+                        response_disposition=None, generation=None):
     """Generate signed URL to provide query-string auth'n to a resource.
 
     .. note::
-      If you are on Google Compute Engine, you can't generate a signed URL.
-      Follow https://github.com/GoogleCloudPlatform/gcloud-python/issues/922
-      for updates on this. If you'd like to be able to generate a signed URL
-      from GCE, you can use a standard service account from a JSON file
-      rather than a GCE service account.
+
+        If you are on Google Compute Engine, you can't generate a signed URL.
+        Follow `Issue 922`_ for updates on this. If you'd like to be able to
+        generate a signed URL from GCE, you can use a standard service account
+        from a JSON file rather than a GCE service account.
+
+    See headers `reference`_ for more details on optional arguments.
+
+    .. _Issue 922: https://github.com/GoogleCloudPlatform/\
+                   gcloud-python/issues/922
+    .. _reference: https://cloud.google.com/storage/docs/reference-headers
```

Comment:
Yeah me too, and this at least matches what `gcloud-node` does.

I'm curious if @jgeewax has any opinion here